packages/core-data/src/entities.js

```diff
@@ -360,13 +360,15 @@ async function loadPostTypeEntities() {
 					 *
 					 * @param {import('@wordpress/sync').CRDTDoc}    crdtDoc
 					 * @param {import('@wordpress/sync').ObjectData} editedRecord
+					 * @param {import('@wordpress/sync').Origin}     origin
```

Comment:
@chriszarate I ended up not actually using this `origin` parameter, newly piped into `getChangesFromCRDTDoc`. I expected to need it when determining if a change came from undo manager versus elsewhere, but the eventual strategy used here didn't need it. I'm keeping it in for now because it wasn't too hard to include this information, and I could see the origin of changes in `getPostChangesFromCRDTDoc()` being useful in the future.

Happy to remove the origin as a parameter if you disagree.

Reply:
Let's remove if it's unused. Can always add back when needed